File: online judge.kaner murillo/python/12996.py
#https://onlinejudge.org/external/129/12996.pdf
#12996

from sys import stdin

casos = int(stdin.readline())

for i in range(1,casos+1):

    persona = [int(x) for x in stdin.readline().split()]

    mangos = [int(x) for x in stdin.readline().split()]
    mangosComer = [int(x) for x in stdin.readline().split()]

    ans = True
    contador = 0
    n = 0

    while n < persona[0] and ans == True:

        if contador>persona[1] or mangos[n] > mangosComer[n]:
            ans = False

        contador+=mangos[n]

        n+=1
    
    if ans == True and contador<=persona[1]:

        print(f"Case {i}: Yes")
    
    else:

        print(f"Case {i}: No")

File: online judge.kaner murillo/c++/12996.cpp
//https://onlinejudge.org/external/129/12996.pdf
//12996

#include<cstdio>
#include<iostream>

using namespace std;

int arr[11];

int main(){

    int cases, j, i, num, sum1, flag, size, maxi;

    scanf("%i", &cases);

    for(i = 1; i<= cases; i++){

        scanf("%i %i", &size, &maxi);
        sum1 = 0;
        flag = 1;

        for(j = 0; j<size; j++){
            scanf("%i", &arr[j]);
            sum1+=arr[j];
        }

        for(j = 0; j<size; j++){
            scanf("%i", &num);
            if(num < arr[j]){
                flag = 0;
            }
        }
        printf("Case %i: ", i);
        if(sum1 > maxi || !flag){
            printf("No\n");
        }
        else{
            printf("Yes\n");
        }

    }

    return 0;
}